tests/analysis.spec.ts

```diff
@@ -386,4 +386,25 @@ describe('Functional test of analysis', () => {
       expect(bundle).toBeTruthy();
     });
   });
+
+  describe('analyzeScmProject', () => {
+    it('should successfully analyze SCM project', async () => {
```

Comment:
The mocks in that other test don't affect these tests, it doesn't fail because the params are not empty, just dummy values, so the report will still init as validation passes, but eventually fail once processing.
i.e. `initReport` will work fine with those dummy values, but once the job actually starts and tries to validate the project ID and/or commit ID it will fail, so only `getReport` would get the failures (if it weren't mocked, that is).